examples/nlp/scripts/multiwoz/process_multiwoz.py

```diff
@@ -0,0 +1,398 @@
+# =============================================================================
```

Comment:
I think scripts should start with a shebang:

#!/usr/bin/python

Reply:
Added to the file.